lighthouse-core/report/html/report-styles.css

```diff
@@ -1543,4 +1543,46 @@
   100% { opacity: 1;  filter: drop-shadow(1px 0px 1px #aaa) drop-shadow(0px 2px 4px hsla(206, 6%, 25%, 0.15)); pointer-events: auto; }
 }
 
+/* Element screenshot */
+.lh-element-screenshot {
+  position: relative;
+  overflow: hidden;
+  float: left;
+  margin-right: 20px;
+  cursor: zoom-out;
+}
+.lh-node .lh-element-screenshot {
+  cursor: zoom-in;
+}
+.lh-element-screenshot__content {
+  overflow: hidden;
+}
+.lh-element-screenshot__image {
+  outline: 2px solid #777;
+  background-color: white;
+  background-repeat: no-repeat;
+}
+.lh-element-screenshot__mask {
+  position: absolute;
+  background: #555;
+  opacity: 0.8;
+}
+.lh-element-screenshot__element-marker {
+  position: absolute;
+  outline: 2px solid rgb(211, 232, 75);
```

Comment:
hoist this to a css variable?

Reply:
yup. also like using a color that's defined in a variable.

lime-400 from material colors seems like a close match. `hsl(66, 70%, 61%)`